modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -1168,36 +1239,56 @@ protected function lib_mix($args) {
 	}
 
 	protected function lib_contrast($args) {
-		if ($args[0] != 'list' || count($args[2]) < 3) {
-			return array(array('color', 0, 0, 0), 0);
-		}
+	    $darkColor  = array('color', 0, 0, 0);
```

Comment:
The changes here and on the following lines now have mixed spaces and tabs.  Boo.